docs/extensions.rst

```diff
@@ -0,0 +1,23 @@
+Extensions
```

Comment:
An idea: maybe we hide these 'frameworkie' kind of files that have little value for browsing on github in a separate subdirectory (docs/autodoc or something?). That way, it would be easier to see the development.md and other browsable files easier?

Reply:
sphinx autodoc assumes a `docs` directory, but that is configurable.

That said, if we fold the .md files into readthedocs, then all files under `docs/` would become the same "type", right?

Reply:
@selitvin Would it be OK if I address this in a separate PR where I make all .mds rst files?